index.html

```diff
@@ -2757,10 +2757,28 @@ <h3><dfn>Writing or pushing content</dfn></h3>
             </ul>
           </li>
           <li>
-            If |record|'s <a>isLocal</a> is `true` and |record| is not a payload
-            to another <a>NDEF record</a>, reject |promise| with a {{TypeError}}
-            and abort these steps.
-          </li>
+            If |record|'s <a>isLocal</a> is `true`:
```

Comment:
Looks like you moved this to the front, but the validate local type steps were written with the old algorithm in mind. If we check local types up-front, we need to update/change the validation algorithm as well.

Reply:
Hmm, on a second look, the current validation algorithm should fare pretty well here as well.